cliquet/tests/test_views_heartbeat.py

```diff
@@ -16,3 +25,17 @@ def test_returns_database_false_if_ko(self, *mocked):
             mock_instance.return_value = False
         response = self.app.get('/__heartbeat__', status=503)
         self.assertEqual(response.json['database'], False)
+
+    @mock.patch('cliquet.cache.redis.Redis.ping')
+    def test_returns_cache_false_if_ko(self, *mocked):
+        for mock_instance in mocked:
+            mock_instance.return_value = False
+        response = self.app.get('/__heartbeat__', status=503)
+        self.assertEqual(response.json['cache'], False)
+
+    @mock.patch('requests.get')
+    def test_returns_oauth_false_if_ko(self, *mocked):
+        for mock_instance in mocked:
+            mock_instance.side_effect = requests.exceptions.HTTPError()
+        response = self.app.get('/__heartbeat__', status=503)
+        self.assertEqual(response.json['oauth'], False)
```

Comment:
I added it here: https://github.com/mozilla-services/cliquet/pull/184/files#diff-f019564cfafcf39e7396d1e84e019e1dR43